src/styles/blocks/card.css

```diff
@@ -0,0 +1,93 @@
+.card {
+  width: 200px;
+
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  font-weight: 400px;
```

Comment:
1 - font-weight it's just a value, without unit dimensions, so it doesn't work now (check in dev tools)
2 - it's a default value, so can be removed